types/sinon/sinon-tests.ts

```diff
@@ -167,8 +167,8 @@ function testXHR() {
 }
 
 function testClock() {
-    let clock = sinon.clock.create(1000);
-    clock = sinon.clock.create(new Date());
+    let clock = sinon.clock.create(1000) as sinon.SinonFakeNodeClock;
+    clock = sinon.clock.create(new Date()) as sinon.SinonFakeNodeClock;
```

Comment:
did you put these casts here so you can test that a clock is assignable to a `SinonFakeNodeClock` (and that it exists)?

would be nice to keep the old code (in addition if needed) so we can make sure the default types also work

Reply:
The old test is no longer working due to contravariance. It's now both a browser and node clock unless explicitly casted to.

Reply:
what failure does it cause? you reverted some of your changes so maybe it'll work fine now? (you removed your change to the clock type)